### PR TITLE
Feature/4065 display bank partners queries

### PR DESCRIPTION
**Pull request by cocymsc1986**
Display bank partners using the gql server to return data, including filtering and pagination (FE pagination not yet implemented)

### CODE REVIEW

File: packages/client/src/app/pages/Banks/components/BankPartnersList.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          // locked={deleting}
+          show
+          bodyText="Are you sure you want to delete the bank partner?"
+          okButtonText="Yes"
+          cancelButtonText="No"
+          testId="bank-partner-delete-dialog"
+          // onClick={() => {
+          //   setDeleting(true);
+          //   onDeleteBankPartner(partnerToDelete);
+          // }}
```
Comment (cocymsc1986) [inline packages/client/src/app/pages/Banks/components/BankPartnersList.js:57]: intentionally left until we implement delete functionality

File: packages/client/src/app/pages/Banks/components/BankPartnersList.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      />
+      {!!partnerToDelete && (
+        <Dialog
+          // locked={deleting}
+          show
+          bodyText="Are you sure you want to delete the bank partner?"
+          okButtonText="Yes"
+          cancelButtonText="No"
+          testId="bank-partner-delete-dialog"
+          // onClick={() => {
```
Comment (andrewchaa) [inline packages/client/src/app/pages/Banks/components/BankPartnersList.js:54]: Do we need this piece of code?

**Review by andrewchaa — APPROVED** [review]
LGTM but we can delete the commented out code if it's not needed

File: packages/client/src/app/pages/Banks/components/BankPartnersList.js (hunk 2 above)
Comment (cocymsc1986) [reply packages/client/src/app/pages/Banks/components/BankPartnersList.js:54]: I had left it in intentionally as I know this functionality will be added

**Pull request merged**